### DISCUSSION TITLE
web interface not available and couple of other issues

### DISCUSSION THREAD

**Discussion by apopovski72** [AzuraCast/AzuraCast#7199] [General]
Hello Everyone,
Few issues so I appreciate any thoughts on resolving them. Azuracast running as a docker instance.  

1.After some time web interface stops responding while the radio streams continue to work.   
If I restart container the issue remains.  If restart nginx in the  container the issue remains.  If I restart docker service on the host the web interface is back.

2.Everytime when I would like to connect a radio stream I have to restart the broadcast service.  For example stream is working then it stops then I need to restart broadcast service for the stream to connect. 

Any advise how to resolve the issues or how to debug them?

**Comment by BornKillaz** [comment]
Hi. 

I'm also new to AzuraCast, but how's your CPU and RAM values?

In AutoDJ, what features do you have enable?

And the logs? Have you look at them? But don't [just look at them](https://youtu.be/EF8GhC-T_Mo?si=dpdOQ1UcPncFbAjR) as Ed Bassmaster would do, but also post them here and that probably would be a huge help.

🙂

**Comment by apopovski72** [comment]
Hello BornKillaz,

I just figure out the issue with the first issue with the web interface.  I added the container web interface port and restarted csf and after that the web interface returned.  Just I am not sure how previously it was working and then stopping while with docker deamon restart it was returning back .  But its working now hopefully it will stay :).

Second issue is still there  but I will test it over the wknd.  There are also other small issues like station offline while its working etc.

Regarding your questions CPU/RAM usage is like 5%.   Autodj is enable but not used so I will disable it now.  As for the logs nothing really shows except autodj errors.

Thank you for the help.

**Comment by apopovski72** [comment]
Here is a new problem.  :)
I just clicked     " Do not use an AutoDJ service "   and the following error appeared n Object(App\Entity\Station): The port 10002 is in use by another station.   There is no station with that port.

**Comment by BornKillaz** [comment]
Looks like you have a port issue. What ports have you configure for
AzuraCast? The default ones? Have you tried to temporarily disable the
firewall to check it still happens?

Also, check what is using that port

# netstat -tulpn | grep 10002

**Comment by apopovski72** [comment]
Hello BornKillaz,

Sorry for the slight delay and thank you for the help.  I just tried checking that port and the result was empty meaning no such port is used.  So I guess is something within azuracast.

**Reply by BornKillaz** [reply]
Be sure to set the ports you want to use:

```
cd /var/azuracast
./docker.sh update-self
./docker.sh change-ports
```

Then whitelist those ports in the firewall.

Check if it solves the problem.

If not, completely disable the firewall.

Check again.

If not, try to reinstall AzuraCast...

Check again.
